src/transformers/utils/coco_classes.py

```diff
@@ -0,0 +1,94 @@
+# COCO object detection id's to class names
+id2label = {
```

Comment:
If this is only used in the conversion script I would put it there

Reply:
I know but IRRC @sgugger suggested to put these into a separate file to make the conversion script short and concise (same was done with `imagenet_classes.py` in the ViT PR)

Reply:
Ah, my bad then!